os/fs/smartfs/smartfs_smart.c

```diff
@@ -1917,6 +1917,7 @@ int smartfs_rename(struct inode *mountpt, const char *oldrelpath, const char *ne
 	struct smartfs_entry_s newentry;
 	mode_t mode;
 	uint16_t type;
+	struct smart_read_write_s readwrite;
```

Comment:
please wrap this with sector buff config.

Reply:
yes, done